components/locale_core/src/langid.rs

```diff
@@ -26,8 +26,10 @@ use writeable::Writeable;
 /// At the moment parsing normalizes a well-formed language identifier converting
 /// `_` separators to `-` and adjusting casing to conform to the Unicode standard.
 ///
-/// Any bogus subtags will cause the parsing to fail with an error.
-/// No subtag validation is performed.
+/// Any syntactically invalid subtags will cause the parsing to fail with an error.
+///
+/// This operation normalizes syntax only. No legacy subtag replacements is performed.
+/// For data validation and subtag canonicalization, see `LocaleCanonicalizer`.
```

Comment:
Why "data validation" instead of "validation"? Why "subtag canonicalization" instead of "canonicalization? The list above doesn't say "well-formed", "data-valid", and "subtag-canonical".

Reply:
I'm ok with changing it to:

> This operation normalizes syntax to well-formed. No legacy subtag replacements is performed.
> For validation and canonicalization, see `LocaleCanonicalizer`.

Reply:
Done.